app/components/bali/page_header/component.rb

```diff
@@ -5,20 +5,39 @@ module PageHeader
     class Component < ApplicationViewComponent
       attr_reader :options
 
-      renders_one :title, ->(text = nil, **options, &block) do
-        options[:class] ||= 'title is-1 mb-0'
-        text.present? ? tag.h1(text, **options) : tag.div(&block)
+      renders_one :title, ->(text = nil, type: :h3, **options, &block) do
```

Comment:
I think `tag` is a better name for this argument

```suggestion
      renders_one :title, ->(text = nil, type: :h3, **options, &block) do
```